src/IO/WriteBufferFromS3.cpp

```diff
@@ -242,7 +253,13 @@ WriteBufferFromS3::~WriteBufferFromS3()
     // That destructor could be call with finalized=false in case of exceptions
     if (!finalized)
     {
-        LOG_ERROR(log, "WriteBufferFromS3 is not finalized in destructor. It could be if an exception occurs. File is not written to S3. {}.", getLogDetails());
+        LOG_INFO(log,
```

Comment:
Why the level is `INFO` now? It's still quite bad situation.

Reply:
Actually this is common case. I have made it as `ERROR` because I wish to see what tests affected. But all destructors do finalise as a result no one test prints it.
Now when I removed finalize call from S3 related buffers d-tors I see a lot of tests affected by this error. And it is just OK, just message. `INFO` level is enough for debug.